src/main/java/de/zalando/aruha/nakadi/service/Registry.java

```diff
@@ -0,0 +1,32 @@
+package de.zalando.aruha.nakadi.service;
+
+import com.google.common.collect.ImmutableList;
+import de.zalando.aruha.nakadi.domain.PartitionResolutionStrategy;
+import de.zalando.aruha.nakadi.partitioning.PartitioningStrategy;
+
+import java.util.List;
+
+public class Registry {
```

Comment:
But in general the idea was that this registry will also contain stuff like ValidationStrategies and Enrichment strategies. So it really can be `StrategiesRegistry`. I will rename.